Update settings.reg with propositions

Need74 opened the pull request:
Good morning,

I have just proposed a modification to the settings.reg file (for Windows 11) which allows the ExplorerPatcher settings to be displayed in a slightly different form. 
I also modified settings10.reg (for Windows 10) in the same spirit. 
Part of the proposal is purely aesthetic and concerns line tabulations for alignment or line breaks to create paragraphs making it easier to identify the functions concerned. 
Highlighting Internet links with a double ↗ ↗ makes it possible to differentiate between internal links which operate outside the network and those which require connection. 
The second part concerns the creation of new links, slight text modifications and the proposal of conditional displays of particular parameters. 
It is quite obvious that this is only a suggestion because at least for the aesthetic part it is a matter of taste and as the creator of the program you are the only decision maker. 
For the rest, I have available the translated French language version of the two files Settings.reg and Settings10.reg and I would find it interesting to propose in the ExplorerPatcher project a Translate folder with a sub-folder per language for all versions of these two Settings files so that everyone can recreate the dll files in their language from the source code. 
You would then need to add a link in the Update tab which encourages the user to participate in the translation of EP into their own language without you having to manage anything on your side. 

I have added the following points to each relevant tab: 

① Taskbar tab
- Text Shift
- Removal of unnecessary white space
- Creation of new Text link at the bottom of the page [ ;y Detailed information on how to configuring the Taskbar on the ExplorerPatcher site ]
- Internet link identification by double arrows ↗ ↗
- Creation of the new link at the bottom of the page [ ;https://github.com/valinet/ExplorerPatcher/wiki/All-features ] ② System tray tab
- Line break
- Editing text
- Line break
- Editing text
- Added << >> signs
- Correction 2 double points ::
- Line break 

③ File Explorer tab
- Editing link text
- Added double arrow ↗ ↗
- Moving the link text and the link to the bottom of the page
- Line break
- Editing text
- Line break
- The link text and the link are clearly present at the bottom of the page 

④ Start menu tab
- Text Shift
- Line break
- Editing text
- Editing text
- Line break 

⑤ Window switcher tab
- Text Shift
- Editing text
- Creation of a condition (you will of course have to create it as a variable in the dll)
- Text modification (which makes you want to test the SWS function)
- End of condition
- Editing text
- Editing text
- Line break
- Editing link text
- Added double arrow ↗ ↗ 

⑥ Weather tab
- Creation of a condition (you will of course have to create it as a variable in the dll)
- Added text [ ;t To see all the page options, activate the line (Show Weather on the taskbar)! ] (in order to make you want to test the weather display function)
- End of condition
- Line break
- Moving link text
- Added double arrow ↗ ↗
- Moving the link 

⑦ Spotlight tab
- No changes in this tab 

⑧ Other tab
- Editing text
- Editing text
- Line break
- Added << >> signs
- Added << >> signs
- Added << >> signs
- Added << >> signs 

⑨ Update tab
- Text Shift
- Editing text
- Editing text
- Line break
- Added double arrow ↗ ↗
- Line break
- Editing link text
- Added double arrow ↗ ↗ 

⑩ Advanced tab
- Editing text to link text
- Deleting link text
- Added double arrow ↗ ↗
- Line break
- Passage in two lines
- Line break
- Passage in two lines
- Passage in two lines
- Line break
- Creation of link text
- Added double arrow ↗ ↗
- Creation of a link 

⑪ About tab
- Text Shift
- Text Shift
- Line break
- Addition of double arrows ↗ ↗ to the 4 Internet link texts that follow
- Line break
- Editing link text
- Added double arrows ↗ ↗
- Editing text
- Editing text
- Added the text line [ :t It is necessary to restart File Explorer for settings marked with a star (*) to take effect. ] which will undoubtedly allow many users to save time in getting started with the software. 

Best regards

Amrsatrio commented:
Hi, sorry but I feel like it's better to keep the current design as it is. I have some points to say:
1. From what I've experienced, <<text>> is valid in French (at least through my Canadian friend) but it's not natural in English. So does nouns that have hyphens in it.
2. We are currently in the process of making a new UI that is localizable and still uses the usual Valinet's .reg file format. So, if you want to make it available in French, I will send you the English .rc files (that I have consolidated myself) through Discord. You can join our server through the README. The old menu will stay, only when the new menu cannot run due to whatever reasons.

![image](https://github.com/valinet/ExplorerPatcher/assets/20662640/775f1195-c253-4cba-9a55-c5d16fdc9870)

Need74 commented:
Hi Amr Satrio

Thank you for your response which shows me that even if my approach was clumsy it had the merit of raising questions.
I humbly recognize that my action was far too ambitious, especially for a first contact.

I have been using Explorer Patcher for almost two years on my various PCs in a personal context and I have been passionate about computers since 1987 with Windows 2.0 and MS DOS 3.2 and my knowledge is only limited despite everything because I am in no way programmer.
When I saw that Explorer Patcher was exclusively in English but that it was open source I worked on its translation for purely personal use.

As I am very curious, I tried to understand how the two Settings.reg files worked and I was interested in making the understanding more obvious to an average user, including my friends to whom I introduced EP , and who like me is not a computer scientist.
Lately I told myself that I had to share my work so that new users could save time in mastering the software.

I should not have mixed the work of formatting and the actual work of defects or lack of information and I see today that I have not achieved my primary goal, to share my thoughts.
I have noted your thoughts on the <>, the double arrows as well as the hyphens - and I agree with your opinion because it is obvious that it is in an international spirit that the approach must be taken.

Thank you for the link to ExplorerPatcher-jp which will perhaps allow me to share my French translation.
You offered to send me the .rc files in English and I thank you because indeed, without being able to do programming I should be able to use this base to integrate the French translation.

For the rest, I will certainly make a new proposal in the coming days to update the .reg files because at least 3 or 4 points that I had raised seem important to me and they were not noted in your email.
It is not uncommon for software to be discovered by chance and downloaded from a computer magazine or other relay site. It therefore seems important to me to put a link on the taskbar tab which is the first to open towards the page:
“;y Detailed information on how to configure the Taskbar on the ExplorerPatcher page”
“;https://github.com/valinet/ExplorerPatcher/wiki/All-features”

Concerning the Simple Windows Switcher, it also took me a while to discover this very well designed tool which I no longer do without today. This is why it seemed important to me to include a text to discover it when it was not selected.
To achieve this result, as I indicated in my first attempt, it would be necessary to create a variable “!IsSWSEnabled” (in the same way as Mr. Valinet did for the conditional display of options when SWS is selected but in fact for the opposite condition) in the binary of the .dll. Then simply enter in settings.reg the sentence you want to see displayed if the SWS is not selected and creating the start of the condition with ;s and closing the condition with ;g after the sentence:
;s No_WindowSwitcher_SWS !IsSWSEnabled
;t Some settings only apply to the Simple Window Switcher!
;g No_WindowSwitcher_SWS
Unfortunately, I am not competent to create this variable because I am naïve in programming and my .reg file is therefore inoperable as it stands.

Concerning the Weather, it is the same because if the person has not had the curiosity to select the Weather the page remains blank and it is a good choice for fluidity but as a result there is no awakening of the curiosity and therefore the goal of customizing its Windows interface is not achieved.
It should be noted that I discovered the function by chance because in settings10.reg (for Windows 10) the display of the weather parameters is permanent and not cleared when the choice is “no”. In addition, under Windows 11 the weather tab is only available if you opt for the old Windows 10 start menu and from this point of view there is missing information in the Start Menu tab to mean that the weather is only available with the Windows 10 type Start Menu choice.
I also suggested adding a web link to the weather GitHub page.

It also took me a while to understand the fact that the options have a star at the end of the line and it seems essential to me to constantly remind you of this constraint at the bottom of the page on all tabs.
Added the line of text at the end of the .reg file, after the ;f fixing the end of the tabs: “:t It is necessary to restart File Explorer for settings marked with a star (*) to take effect. » which will undoubtedly allow many users to save time in getting started with the software
I hope I don't have been too long or pretentious but making my small contribution to the world of free software is part of my daily aspirations.
Sincerely.

Denis G
